rpm/netavark.spec

```diff
@@ -88,6 +88,13 @@ Its features include:
 * Support for IPv4 and IPv6
 * Support for container DNS resolution via aardvark-dns.
 
+%package tests
+Summary: Tests for %{name}
+Requires: %{name} = %{epoch}:%{version}-%{release}
+
+%description tests
+%{summary}
```

Comment:
We had this discussion recently about the podman RHEL rpm that this is not supported. So we should make sure if this ever gets top RHEL we do not ship it outside of the gating env if possible (i.e. follow whatever we decided to do with the podman rpm).

Outside of the bats test we do not support these examples either upstream. So maybe add the description here that these are test only binaries for gating and not recommended/supported for any other use.